### ISSUE TITLE
Team compass docs aren't rendering properly?

### ISSUE THREAD

**Issue by sgibson91** [2i2c-org/team-compass#146]
# What isn't working?

I was going through the team-compass site and noticed that [this page](https://2i2c.org/team-compass/practices/coordination/) doesn't match up with the [Markdown](https://github.com/2i2c-org/team-compass/blob/main/practices/coordination.md). It's the same up until the "Weekly team sync" section and then there's a "What about stuff that shouldn't be public?" section on the live site which is missing from the markdown, and sections on monthly team meetings and time off that are not on the live site. Is this intentional or a bug?

# How should it work instead?

The live site should reflect the Markdown in this repo. Anything that was intentionally not rendered should perhaps be located and documented elsewhere?

# Actions to improve this?

- [ ] Decide if this is intentional or a bug
- [ ] If intentional: migrate the non-rendered docs somewhere more appropriate and document that
- [ ] If bug: dive into the Sphinx build to find the problem

**Comment by choldgraf** [comment]
ah you found a bug! We used to use github pages for our docs, and then switched to ReadTheDocs. As part of that we set up redirects for sub-directories, so that `2i2c.org/pilot` redirected to `pilot.2i2c.org`. We apparently forgot to do this for team-compass!

I just pushed a change to add a `team-compass` entry for our website, so `2i2c.org/team-compass` should now re-direct to `team-compass.2i2c.org`. I also deleted the `gh-pages` branch from `team-compass` since we were no longer using it.
